docs/source/services/timelock_service/migration.rst

```diff
@@ -31,8 +31,11 @@ Automated Migration
    is at least 0.253.2.
 2. (Optional) Take a fresh timestamp from your AtlasDB services, using the fresh timestamp CLI or the
    ``/timestamp/fresh-timestamp`` endpoint. This step is not strictly required, but may be useful for verification.
-3. Add the :ref:`Timelock client configuration <timelock-client-configuration>` to your service.
-4. Starting/re-starting the service will automatically migrate the service.
+3. Shut down all nodes of your service (in addition to any services hitting your embedded timelock endpoints). These must
```

Comment:
Not sure the in addition part is necessary? The purpose of shutting down service nodes here is that we don't want the service to continue to give timestamps when another node has fast forwarded timelock and started using it. For "embedded" clients, they will just fail to connect to timestamp and lock... and they won't automatically migrate to timelock either (they may not even know timelock exists).

Actually, I think we should add a warning: if other services are dependent on the timestamp and lock service exposed by an AtlasDB, steps should be taken to move them to timelock before this service is migrated to timelock, and for those services a manual migration will be necessary (seeing as they must be using remote timestamp and lock blocks, and/or misusing timelock configuration to point to those nodes).